src/green/InterfacesImpl.hpp

```diff
@@ -111,17 +111,16 @@ class LnTransformedRadial {
    *  \param[out] drhodr state vector holding the first and second derivative
    *  \param[in] r position on the integration grid
```

Comment:
Update parameter name from `r` to `y` in documentation comment. This should have been printed out as a warning, if you use Clang.
Also can you make sure the description of the class reflects what's being done with the differential equation? That is, can you make clear that we're transforming both the function _and_ the domain?